modules/metering-reports.adoc

```diff
@@ -262,6 +262,37 @@ spec:
   reportingEnd: "2019-07-31T00:00:00Z"
 ----
 
+[id="metering-expiration_{context}"]
+== expiration
+
+Add the `expiration` field to set a retention period on a scheduled metering Report. You can avoid manually removing the Report by setting the `expiration` duration value. The retention period is equal to the Report `creationDate` plus the `expiration` duration. The Report is removed from the cluster at the end of the retention period if no other Reports or ReportQueries depend on the expiring Report. Deleting the Report from the cluster can take several minutes.
```

Comment:
@bentito @pruan-rht Added the last sentence per Brett's suggestion:

> Deleting the Report from the cluster can take several minutes.

Reply:
👍

Reply:
@lbarbeevargas how about this instead?  ```Deleting the Report from the cluster can take several minutes from the expected expiration time```?